fuzzers/libfuzzer_stb_image/src/main.rs

```diff
@@ -68,7 +68,7 @@ fn fuzz(corpus_dirs: &[PathBuf], objective_dir: PathBuf, broker_port: u16) -> Re
 
     // Create an observation channel using the coverage map
     // We don't use the hitcounts (see the Cargo.toml, we use pcguard_edges)
-    let edges_observer = unsafe { std_edges_map_observer("edges") };
+    let edges_observer = unsafe { std_edges_map_observer("edges").with_tracking::<true, false>() };
```

Comment:
just `.tracking`?

Or maybe a different word than tracking alltogether :D. 
Also, `true`, `false` is not super usable but I guess that's the best we can do here? (it's hard to figure out what they mean)

Reply:
Could we impl a .tracks_novelties() fn that returns the same type with a `true` here?

Reply:
Yes, sadly we are rather restricted when it comes to const generics here.

Reply:
We can certainly make separate `tracks_indices`/`tracks_novelties` methods.

Reply:
Done, check the updated changes.